draft-ietf-bidding-and-auction-services.md

```diff
@@ -778,6 +778,11 @@ response = {
   ; If not present, map as Null.
   ? buyerAndSellerReportingId: tstr,
 
+  ; Optional SelectedBuyerAndSellerReportingId of the winning Ad
+  ; Maps directly to https://wicg.github.io/turtledove/#server-auction-response-buyer-and-seller-reporting-id. (TODO: update before merge)

```

Comment:
update after W3C spec's [pull request](https://github.com/WICG/turtledove/pull/1348) that defines the new field is merged.

Reply:
done.